torchtune/_cli/download.py

```diff
@@ -85,18 +87,6 @@ def _add_arguments(self) -> None:
             default=None,
             help="Directory in which to save the model. Defaults to `/tmp/<model_name>`.",
         )
-        self._parser.add_argument(
-            "--output-dir-use-symlinks",
-            type=str,
-            required=False,
-            default="auto",
-            help=(
-                "To be used with `output-dir`. If set to 'auto', the cache directory will be used and the file will be"
-                " either duplicated or symlinked to the local directory depending on its size. It set to `True`, a"
-                " symlink will be created, no matter the file size. If set to `False`, the file will either be"
-                " duplicated from cache (if already exists) or downloaded from the Hub and not cached."
-            ),
-        )
```

Comment:
deprecated. Its a no-op.

Reply:
Should we add a deprecation warning rather than removing?

Reply:
good question! There is already a deprecation warning on huggingface side, so i think it should be ok to remove. Let me know if you disagree.